lib/features/give/pages/qr_code_scan_page.dart

```diff
@@ -73,58 +44,77 @@ class _QrCodeScanPageState extends State<QrCodeScanPage> {
     final size = MediaQuery.sizeOf(context);
     final locals = context.l10n;
     final userGuid = context.read<AuthCubit>().state.user.guid;
-    return Scaffold(
-      appBar: AppBar(
-        leading: const BackButton(),
-        title: Column(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: [
-            Text(locals.giveDifferentScan),
-            Text(
-              locals.giveDiffQrText,
-              style: Theme.of(context).textTheme.bodyMedium,
-            ),
-          ],
-        ),
-        toolbarHeight: size.height * 0.1,
-      ),
-      body: BlocConsumer<GiveBloc, GiveState>(
-        listenWhen: (previous, current) => previous != current,
+    return BlocProvider.value(
```

Comment:
Any reason you're using provider here as we are just getting rid of the providers? I see that you defined the cubit in getit so I don't think you need it?